job_service/adapter/maintenance_db.py

```diff
@@ -30,7 +30,7 @@ def set_status(status_request: MaintenanceStatusRequest):
             "timestamp": str(datetime.now()),
         }
         doc_id = maintenance.insert_one(document)
-        return {"_id": doc_id, **document}
+        return {**document}
```

Comment:
you can just do a `return document` it will do the same 👍🏻

Reply:
Late on a friday evening ... OMG